heron/common/src/cpp/network/connection.cpp

```diff
@@ -75,12 +75,10 @@ Connection::~Connection() {
   delete[] mIOVector;
 }
 
-sp_int32 Connection::sendPacket(OutgoingPacket* packet) { return sendPacket(packet, NULL); }
```

Comment:
I think it is better keep the method with call back param to let the connection.h/cpp be general. There is method without call back param, which can be adopted when call back is not needed.